scene/gui/text_edit.cpp

```diff
@@ -3675,7 +3675,6 @@ void TextEdit::_push_current_op() {
 		return; // do nothing
 
 	if(next_operation_is_complex) {
-		current_op.chain_forward=true;
```

Comment:
What would be the consequence of removing this line, apart from fixing the crash? Isn't this `chain_forward` variable used for something specific?